wallet-sdk/src/main/kotlin/org/stellar/walletsdk/anchor/Anchor.kt

```diff
@@ -82,6 +91,25 @@ internal constructor(
     return Interactive(this, httpClient)
   }
 
+  /**
+   * Creates new transaction watcher
+   *
+   * @param pullDelay pull interval in which requests to the Anchor are being made.
```

Comment:
I think you mean `poll`, not `pull`, right?

Reply:
Yep